react/features/device-selection/components/DeviceSelection.js

```diff
@@ -384,7 +398,7 @@ class DeviceSelection extends AbstractDialogTab<Props, State> {
                 label: 'settings.selectMic',
                 onSelect: selectedAudioInputId =>
                     super._onChange({ selectedAudioInputId }),
-                selectedDeviceId: this.props.selectedAudioInputId
+                selectedDeviceId: this.state.previewAudioInputId
```

Comment:
What about calling `getDeviceId` here instead of storing a separate value for state?

Reply:
Does it look ok now?